### PR TITLE
Brullbar

### PR DESCRIPTION
**Pull request by Eagletanker**
## About the PR 

Renames Wendigo's to be Brullbars

## Why's this needed? 

Wendigos are part of First Nations mythology and they dont like people apporpiating it. This PR replaces that with the name Brullbar, which means Howler Bear. No changelog needed. Umlauts dropped on purpose.

### CODE REVIEW

**Comment by Eagletanker** [comment]
Trying to get help for the type paths. Dont know why they are broken just in those 10 instances

**Comment by Eagletanker** [comment]
cool its still breaking *scream

**Comment by coolbugcoder** [comment]
While I'm fine with changing the name, why Brullbar? I can't find any type of monster named that, and a pale white skinny being doesn't really give the image of a bear to me. I think another name that more reflects its design would be much better.

**Comment by coolbugcoder** [comment]
well, on hindsight, I think the name is decent at keeping the mythical sounding nature of wendigo and also reflects the howling nature, even if I don't think the "bear" part fits super well. Along with that, I can't find a good alternative myself admittedly. I thought Ghoul would work but that's apparently originally part of Arabian folklore and originally was mainly hanging around cemeteries, so that would probs have the same issue as Wendigo. Brullbar is okay even if I think an also mythical or threatening sounding name without the bear part would be better.

**Comment by Eagletanker** [comment]
> While I'm fine with changing the name, why Brullbar? I can't find any type of monster named that, and a pale white skinny being doesn't really give the image of a bear to me. I think another name that more reflects its design would be much better.

Chosen from a list of names. It's purposefully it's own thing as researchers on [REDACTED] found them and screaming bear thing is probably what was gleamed from the few survivors of such attacks. We also have the non bear like space bear

**Comment by coolbugcoder** [comment]
> > While I'm fine with changing the name, why Brullbar? I can't find any type of monster named that, and a pale white skinny being doesn't really give the image of a bear to me. I think another name that more reflects its design would be much better.
> 
> Chosen from a list of names. It's purposefully it's own thing as researchers on [REDACTED] found them and screaming bear thing is probably what was gleamed from the few survivors of such attacks. We also have the non bear like space bear

yeah, it's cool then.

**Review by TTerc — COMMENTED** [review]
I assume icon state names in DMIs need an update too

**Comment by NightmareChamillian** [comment]
This is massively needed change, but I do think the word "brullbar" is a little hard to take seriously, especially with how pants shittingly terrifying they are. If you're going with early European words, I suggest the "werr" or "wiff" roots to better reflect the fact that it's a fucked up angry pale guy who yells and then disembowel you. 

Regardless, this is a much needed step up from using wendigo.

**Comment by Eagletanker** [comment]
> 
> 
> I assume icon state names in DMIs need an update too

Will do

**Comment by coolstation** [comment]
nice, adopting this.

**Comment by Eagletanker** [comment]
> 
> 
> I assume icon state names in DMIs need an update too
Done

**Comment by Kyle2143** [comment]
Is this complete? The name choice is fine,

**Comment by Eagletanker** [comment]
> Is this complete now? The name choice is fine, it's not particularly terrifying, but then most names aren't and I don't think that's important.
> 
> So I'm fine to merge it on that score, but based on the comments and commits here it looked like there were some issues and it hadn't been compiled and tested locally.

Yep, let me clear the merge conflict and then do a last test compile and it should be good to go 
(hopefully :shelterfrog:)

File: code/datums/mapgen/IceMoonGenerator.dm [hunk 1]
```diff
@@ -6,7 +6,7 @@
 	flora_types = list(/obj/stone/random = 10, /obj/decal/fakeobjects/smallrocks = 10)
 	flora_density = 1
 
-	fauna_types = list(/obj/critter/sealpup=15, /obj/critter/wendigo=5, /obj/critter/yeti=1)
+	fauna_types = list(/obj/critter/sealpup=15, /obj/critter/brullbar, p=5, /obj/critter/yeti=1)
```
Comment (pali6) [inline code/datums/mapgen/IceMoonGenerator.dm:9]: This seems incorrect!

**Review by pali6 — CHANGES_REQUESTED** [review]

**Comment by zjdtmkhzt** [comment]
I can't say I find this very intimidating, considering 95% of search results for Brüllbär are for the children's book series with a cute small bear making friends at kindergarten or whatever, but I already didn't find Wendigos very intimidating.
I mean, they are obj critters, you just kinda hit them until they die.

**Comment by Eagletanker** [comment]
*scream I'll fix the conflict when I get home

**Comment by ZeWaka** [comment]
!merge_upstream

**Pull request merged**